### PR TITLE
Update target user profile and use cases in DG

### PR DESCRIPTION
**Pull request by ro4k9**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -285,6 +284,31 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 (For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
 
+**Use case: Mark an applicant as interview**
```
Comment (seanleong339) [inline docs/DeveloperGuide.md:287]: Should it be interviewed?

**Review by seanleong339 — COMMENTED** [review]
Minor typo error

**Comment by e0543517** [comment]
some check style errors that is causing the CI to fail, can look at the details to fix it

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (ro4k9) [reply docs/DeveloperGuide.md]: I will fix the typo.

**Pull request closed without merging**